src/main/java/frc/robot/Constants.java

```diff
@@ -55,6 +55,57 @@
         public static String starboardEncoderName = "Starboard";
         public static String portEncoderName = "Port";
         public static String rosIndexName = "rosIndex";
-		public static int ticksPerMeter = 10000;                // Calibrated 12/16/2019
+    }
+
+    public static final class DrivetrainPID{
+
+        public static int kPIDLoopIdx = 0; // The loop Index
+        public static int kTimeoutMs = 30; // The timeout to wait when writing variables to the motors
+        public static boolean portSensorPhase = true; // The phase of the sensor
+        public static boolean portMotorInvert = true; // The inversion of the motor
+        public static boolean starboardSensorPhase = false;
+        public static boolean starboardMotorInvert = false;
+        public static final Gains kGains = new Gains(0.15,  // kP
+                                                     0.0,   // kI
+                                                     1.0,   // kD
+                                                     0.0,   // kF
+                                                     0,     // kIzone
+                                                     1.0);  // kPeakoutput
+        // Measurements and other
+        public static int ticksPerMeter = 10000;        // Calibrated 12/16/2019
```

Comment:
should probably change this comment to a "TODO" so we remember to come back through and actually calibrate this, since the value is actually for a different drivetrain

Reply:
Added a TODO i think